ramp-database/ramp_database/tests/data/ramp_config_aws_iris.yml

```diff
@@ -3,9 +3,12 @@ ramp:
     event_name: iris_aws_test
     event_title: "iris aws"
     event_is_public: true
-    data_dir: ramp-board/ramp_deployment/ramp-data/iris
-    kit_dir: ramp-board/ramp-engine/ramp_engine/tests/kits
+    data_dir: /tmp/databoard_test/ramp-data/iris
+    kit_dir: /tmp/databoard_test/ramp-kits/iris
     sandbox_dir: starting_kit
+    submissions_dir: /tmp/databoard_test/submissions
+    predictions_dir: /tmp/databoard_test/preds
+    logs_dir: /tmp/databoard_test/log
```

Comment:
maybe a bit dangerous that the system as /tmp folder?

Reply:
hm. I used the exactly the same directories for `submissions`, `preds` and `log` as it is already set for `ramp_config_iris.yml` (ie iris running on conda). `boston_housing` is also using `/tmp` folder. If you have alternative suggestion we should change it for all three